quinn-udp/src/unix.rs

```diff
@@ -164,7 +164,7 @@ fn init(io: SockRef<'_>) -> io::Result<()> {
     #[cfg(any(target_os = "freebsd", target_os = "macos", target_os = "ios"))]
     {
         if is_ipv4 {
-            set_socket_option(&*io, libc::IPPROTO_IP, libc::IP_DONTFRAG, OPTION_ON)?;
+            let _ = set_socket_option(&*io, libc::IPPROTO_IP, libc::IP_DONTFRAG, OPTION_ON);
```

Comment:
@Ralith what do you think, should we only ignore this specific error value?

Reply:
Yeah, and only on the iOS (and perhaps macOS) versions expected to produce it. Otherwise we're liable to miss any similar future hazards, which could lead to degraded performance due to incorrect PMTU behavior.